SConstruct

```diff
@@ -425,6 +457,37 @@ if 'NETX90' in atPickNetxForBuild:
     env_netx90_dbg = env_netx90_default.Clone()
     env_netx90_dbg.Append(CPPDEFINES = [['CFG_DEBUGMSG', '1']])
     elf_netx90_dbg, bin_netx90_dbg, lib_netx90_dbg = flasher_build('flasher_netx90_debug', env_netx90_dbg, 'targets/netx90_dbg', src_lib_netx90, src_main_netx90)
+    
+    cwd = os.getcwd()
+    strKeyRomPath = "keys/keys.xml" # must be copied to repo
```

Comment:
does it even work without keys.xml?

Reply:
Why do we need a key file? We're not building  a signed version. If we were, this would need to further above at line 400.